resources/views/pages/questions/overview.twig

```diff
@@ -21,8 +51,17 @@
                     {% block questions %}
                         {% block faq_text %}
                             {% if has_permission_to('faq.view') %}
-                                <p>{{ __('question.faq_link', [url('/faq')]) | raw }}</p>
+                                {% if not is_admin | default(false) %}
+                                    {{ __('question.faq_link', [url('/faq')]) | raw }}
+                                {% endif %}
+                            {% endif %}
+                        {% endblock %}
+                        {%- block info_text %}
+                            <p>
+                            {% if config('contact_options') %}
```

Comment:
Should be hidden on the "answer questions" page

Reply:
@xuwhite this is still an issue?

Reply:
nopw its not